src/pudl/extract/eia860m.py

```diff
@@ -85,3 +90,51 @@ def append_eia860m(eia860_raw_dfs, eia860m_raw_dfs):
             sort=True,
         )
     return eia860_raw_dfs
+
+
+@asset(
+    required_resource_keys={"datastore", "dataset_settings"},
+)
+def raw_eia860m__all_dfs(context):
```

Comment:
this guy is kinda a hack to mirror some of the behavior in the other excel extractors that use `raw_df_factory`